homeassistant/config_entries.py

```diff
@@ -608,6 +608,7 @@ def async_start_reauth(self, hass: HomeAssistant) -> None:
         flow_context = {
             "source": SOURCE_REAUTH,
             "entry_id": self.entry_id,
+            "title_placeholders": {"name": self.title},
```

Comment:
I think that just this change would be enough together with https://github.com/home-assistant/frontend/pull/11358. `pvoutput` would just work out of the box then.

Reply:
Alright, lets do that. My other brainfart doesn't work out anyways, so this would be the best option.